go-controller/pkg/metrics/ovnkube_controller.go

```diff
@@ -257,6 +261,16 @@ var metricIPsecEnabled = prometheus.NewGauge(prometheus.GaugeOpts{
 	Help:      "Specifies whether IPSec is enabled for this cluster(1) or not enabled for this cluster(0)",
 })
 
+var metricIPsecTunnelState = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+	Namespace: MetricOvnkubeNamespace,
+	Subsystem: MetricOvnkubeSubsystemController,
+	Name:      "ipsec_tunnel_state",
+	Help:      "Specifies whether IPSec Child SA is established for the Geneve tunnel(1) or not(0)"},
+	[]string{
```

Comment:
You want to add an alarm too for this ? Well probably use it downstream so may as well add it upstream?

Reply:
I don't see much of IPsec deployments with u/s, so let's start with adding alert rule in d/s.